src/collections/_documentation/performance/discover/query-builder.md

```diff
@@ -236,115 +196,113 @@ Events have a number of built-in fields as well as custom tags.
 {%- endfor -%}
 </table>
 
-### Syntax
-
-The Query Builder syntax is identical to [Sentry's Search syntax]({%- link _documentation/workflow/search.md -%}). To add a search condition, click on the search input box above the graph, add a search condition, and press "Enter."
-
-### Searching on Aggregated Fields/Tags
-
-You can add search conditions on aggregated fields/tags (for example, `count(id)`) in one of the following ways:
-
-- Exact match: `count(id):99`
-- Upper bounds: `count(id):<99` or `count(id):<=99`
-- Lower bounds: `count(id):>99` or `count(id):>=99`
-- Combination of upper and lower bounds: `count(id):>10 count(id):<20`
+**Syntax**
 
-## Results Table
+The Query Builder syntax is identical to [Sentry's Search syntax]({%- link _documentation/workflow/search.md -%}). After you enter a key field from above or a custom tag, you can use any of the referenced syntax. For example, `count(id)` gives you the number of times an event occurs. This can be written in the following ways: 
 
-The results of the table are generated based on the following components:
+- Exact match (is equal to): `count(id):99`
+- Upper bounds (is less than or equal to): `count(id):<99` or `count(id):<=99`
+- Lower bounds (is more than or equal to): `count(id):>99` or `count(id):>=99`
+- Multiple bounds (is more and less than): `count(id):>10 count(id):<20`
 
-- Selected columns
-    - Columns may either be a known field, or a tag
-    - Columns may have an aggregation function applied to them
-    - Sort order depends on the column selected
-- Search conditions
-- Global selection header (projects, environment, dates)
+**Tag Summary Filters**
```

Comment:
Should these be headings and not bold text in a paragraph?

Reply:
I put this under Search Conditions because the tag summary can help build that out.

Reply:
![Screen Shot 2020-06-22 at 11 21 29 AM](https://user-images.githubusercontent.com/4830259/85322062-92cade80-b47a-11ea-8c5f-70482b5c2c09.png)